backend/src/interfaces/user.ts

```diff
@@ -0,0 +1,8 @@
+export interface user {
+  uid: string;
+  first_name: string | null;
+  last_name: string | null;
+  email: string | null;
+  photourl: string | null;
+  phone_number: string;
```

Comment:
is the idea that users only need to provide phone number and the rest can be null? When I program I try to reduce null values to an absolute minimum, that way you dont have to worry about it.